### ISSUE TITLE
bug in vgg.py

### ISSUE THREAD

**Issue by Abhijit-2592** [yhenon/keras-frcnn#198]
@yhenon,
From the Faster RCNN paper under section 3.1:

> This small (RPN network) network takes as input an n × n spatial window of
> the input convolutional feature map. Each sliding
> window is mapped to a lower-dimensional feature
> (256-d for ZF and **512-d for VGG**, with ReLU [33]
> following). This feature is fed into two sibling fullyconnected
> layers—a box-regression layer (reg) and a
> box-classification layer (cls).

But in your implementation in [vgg.py](https://github.com/yhenon/keras-frcnn/blob/master/keras_frcnn/vgg.py) You have given 256-d. Shouldn't it be 512? or am I missing something?
Even the [original](https://raw.githubusercontent.com/rbgirshick/py-faster-rcnn/master/models/pascal_voc/VGG16/faster_rcnn_end2end/test.prototxt) implementation has 512-d
**The original implementation also uses 2 (0.5) dropout on the final classifier's 2 FC layers. What is your intuition behind not using dropouts in your implementation**?

```
def rpn(base_layers, num_anchors):

    x = Conv2D(256, (3, 3), padding='same', activation='relu', kernel_initializer='normal', name='rpn_conv1')(base_layers) # should be  512-d

    x_class = Conv2D(num_anchors, (1, 1), activation='sigmoid', kernel_initializer='uniform', name='rpn_out_class')(x)
    x_regr = Conv2D(num_anchors * 4, (1, 1), activation='linear', kernel_initializer='zero', name='rpn_out_regress')(x)

    return [x_class, x_regr, base_layers]
```

**Comment by yhenon** [comment]
You are indeed correct on both counts, those are mistakes on my end. I will fix this.

**Comment by yhenon** [comment]
Fixes in 38413c69129673d192f38daa553b98af1008c950 . Thanks for the bug report!
